test/recipe/Baked.Test.Recipe.Service.Application/ConfigurationOverrider/ConfigurationOverriderFeature.cs

```diff
@@ -179,114 +179,122 @@ public void Configure(LayerConfigurator configurator)
 
         configurator.ConfigureAppDescriptor(app =>
         {
-            app.Error = ErrorPage(
-                safeLinks:
-                [
-                    CardLink("/", "Home", icon: "pi pi-home"),
-                    CardLink("/specs", "Specs", icon: "pi pi-list-check"),
-                ],
-                errorInfos:
-                [
-                    ErrorPageInfo(403, "Access_Denied", "You_do_not_have_the_permision_to_view_the_address_or_data_specified" ),
-                    ErrorPageInfo(404, "Page_Not_Found", "The_page_you_want_to_view_is_etiher_deleted_or_outdated"),
-                    ErrorPageInfo(500, "Unexpected_Error", "Please_contact_system_administrator")
-                ],
-                data: Computed(Composables.UseError)
-            );
+            configurator.UsingNewLocale(l =>
```

Comment:
let's make this `UsingLocalization(l => ...` to make nuxt and .net even more similar.

on nuxt side it is `useLocalization()` and `l(...)`, and in here it's `UsingLocalization(...)` and `l(...)`.